README.md

```diff
@@ -84,16 +84,16 @@ When you are architecting an app, try to keep these **key principles** in mind:
 - How can you make it **easy to change**?
 - Can you **anticipate changes** that are likely to occur?
 
->"If you don't develop an architecture, you will get one anyway - and you might not like what you get!"
->-- Northrop, [The Importance of Software Architecture](http://csse.usc.edu/GSAW/gsaw2003/s13/northrop.pdf)
+> "If you don't develop an architecture, you will get one anyway - and you might not like what you get!"
+> -- Northrop, [The Importance of Software Architecture](http://csse.usc.edu/GSAW/gsaw2003/s13/northrop.pdf)
 
 ## Exercise 2: Restructuring A File System
 
 Your file structure is an architectural structure in its own right. Again, it is an _attempt to manage complexity_.
 
 Keeping the Key Principles above in mind, how would you reorganise the files of this repository? (They are intentionally empty files, by the way.)
 
-_**Tip**: if you haven't done so already, you can run `npm install file-icons` so that all your atom icons have icons displaying in the file tree, making them easier to distinguish_
+**Tip**: if you haven't done so already, you can run `websemantics/file-icons-js` so that all your files have icons that display in the file tree, making them easier to distinguish. Atom and VS Code also have extensions that do this e.g. Material Icon Theme.
```

Comment:
I don't think the [file-icons-js](https://www.npmjs.com/package/file-icons-js) package does what this tip says it does. The tip says it adds file icons to your text editor, but it actually seems to be for using file icons in the web browser.

I think the suggestion of installing an npm package should be removed from this section, apparently the ```file-icons``` npm package that was originally mentioned here doesn't exist. 

Also, I think the point that "Atom and VS Code also have extensions" could be a little clearer. Material Icon Theme is a VS Code-only extension, not an Atom package. Maybe add an example of an Atom equivalent? (I use [this one](https://atom.io/packages/file-icons), it has 7 million+ downloads).

@oliverjam @astroash I originally thought this pull request was a [tiny](https://github.com/foundersandcoders/master-reference/blob/master/curriculum-planning/curriculum-update-process.md#minor-or-major-change) change, but am tagging you as reviewers since it seems a bit more substantial now. What do you think?

Reply:
I wonder if the original tip was suggesting to run `apm install file-icons`

https://atom.io/packages/file-icons

I believe that vs code displays file icons by default.

Also if we are making changes to this workshop can we please rename it to Software Architecture? The word "design" is redundant in this context and I feel like it gets mixed up with the Software Design workshop because of this.

Reply:
Yeah VS Code has icons by default, and that feels beyond the scope of this workshop either way. I'd just remove it.

+1 to removing "design" from the name